packages/contracts-bedrock/test/L1/OptimismPortal2.t.sol

```diff
@@ -507,6 +514,24 @@ contract OptimismPortal2_FinalizeWithdrawal_Test is CommonTest {
         assertFalse(optimismPortal2.finalizedWithdrawals(Hashing.hashWithdrawal(_defaultTx)));
     }
 
+    /// @notice Sets the supeRootsActive variable to the provided value.
+    /// @param _superRootsActive The value to set the superRootsActive variable to.
+    function setSuperRootsActive(bool _superRootsActive) public {
+        // Get the slot for superRootsActive.
+        StorageSlot memory slot = ForgeArtifacts.getSlot("OptimismPortal2", "superRootsActive");
+
+        // Load the existing storage slot value.
+        bytes32 existingValue = vm.load(address(optimismPortal2), bytes32(slot.slot));
+
+        // Inject the bool into the existing storage slot value with a bitwise OR.
+        // Shift the bool left by the offset of the storage slot and OR with existing value.
+        bytes32 newValue =
+            bytes32(uint256(uint8(_superRootsActive ? 1 : 0)) << slot.offset * 8 | uint256(existingValue));
+
+        // Store the new value at the correct slot/offset.
+        vm.store(address(optimismPortal2), bytes32(slot.slot), newValue);
+    }
```

Comment:
You should be able to use the `stdStorage` library from forge-std and just do
```solidity
stdstore
  .target(address(optimismPortal2))
  .sig("superRootsActive()")
  .checked_write(_superRootsActive);
```
it handles packed slots

Reply:
Oh nice, way better

Reply:
Alright this didn't work very well so I'm going to keep the function as-is